src/Nakama/Client.cs

```diff
@@ -38,6 +36,14 @@ public class Client : IClient
         /// </summary>
         public const int DefaultPort = 7350;
 
+        /// <summary>
+        /// The default expired datetime used to check session lifetime.
+        /// </summary>
+        public static DateTime DefaultExpiredDateTime = DateTime.UtcNow.AddMinutes(5);
```

Comment:
I think this expression will only be evaluated once (when static members are initialized). Therefore five minutes after app boot, if this value is not reset by the user, the client will try to fetch a new authorization token with every API call. I propose instead storing the five minute `Timespan` in its own variable `SessionTokenRefreshWindow` (or something) and moving the `DateTime.UtcNow` access to each generated API call.

Reply:
That's a good catch. I'll update the code to retain the `TimeSpan` but calculate the `DateTime` at the call site.